src/common/header/components/GitHubButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { IconButton } from '@chakra-ui/button';
+import { AiFillGithub } from 'react-icons/ai';
+
+export function GitHubButton(): JSX.Element {
+  return (
+    <IconButton
+      as="a"
+      href="https://github.com/VikeLabs/courseup"
+      target="_blank"
+      size="sm"
+      colorScheme="none"
+      fontSize="35px"
+      isRound
+      aria-label="Go to GitHub Repo"
```

Comment:
minor nit: Since this opens a new tab, maybe I would say `Open GitHub repo` since it's not technically redirecting

Reply:
that definitely makes it clearer, I've [committed the change](https://github.com/VikeLabs/courseup/pull/257/commits/9c85ba5259e81eaac68ff17a4fc26f3439643131) :)